### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_ScreenReader: The grid data is nor announced correctly.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#11198]
["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS1.3.1;#SQL Azure Data Studio;#STP;#JAWS;#A11ySev2;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
5. Open the Command Palette by pressing CTRL+SHIFT+P (Windows)
appears.
6. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu.
7. Python Dependencies Wizard will appear.
8. Provide any Input file details & Next.
9. Launch screen reader as JAWS and navigate using shift +T

**Actual:**
On navigating to the 3rd column row1 using down arrow key and then using ctrl+alt+5 the screen reader announces column 1 row 3 name existing version and required version.

**Issue not observed with NVDA**

**Expected:**
On navigating to each cell using down arrow key and then using ctrl+alt+5 the screen reader should announce column 3 row 1 required version.

**User Impact:**
Screen reader users will receive the wrong information.

**Attachment link for Reference:**
[11198A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4869562/11198A11y_AzureDataStudioStayHealthy_Notebook_Python.Dependencies.Wizard_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
@fsteffi as you mentioned this seems to be a JAWS only issue, and I don't think there is anything we can do about it. our general guideline from the Accessibility team is: there is no need to optimize for a specific screen reader as long as we are setting the accessibility related properties properly.

**Comment by sspatil123** [comment]
@alanrenmsft I have checked in 1.43.0 Version issue is still 
reproducing
